src/nvim/terminal.c

```diff
@@ -701,7 +701,17 @@ void terminal_receive(Terminal *term, char *data, size_t len)
     return;
   }
 
-  vterm_input_write(term->vt, data, len);
+  // Becatuse libvterm uses VLA, it casues stack overflow when large data is
+  // written at once. To prevent this, the data will be split and written until
+  // libvterm is fixed.
```

Comment:
Do you have a link to the relevant bug?

Reply:
No one has reported a bug in [launchpad](https://bugs.launchpad.net/libvterm) yet. I think the only link to this bug is #16040.